### DISCUSSION TITLE
Filter where clause with custom sql condition

### DISCUSSION THREAD

**Discussion by tqwewe** [prisma/prisma#16861] [Q&A]
Is it possible to filter using `where` with a custom sql condition?

In my case, my SQL would be `WHERE category(stream_name) = 'foo'`, but I'm not sure if this is possible in prisma without writing the entire query with rawSql?

**Comment by nurul3101** [comment]
Hello @tqwewe 👋 
It looks your use case is similar to this Feature Request:

- #5560

It's not possible at the moment, I would recommend you to provide a 👍 to the feature request, so that our product team can prioritise it. At the moment, you would need to resort to Raw Query.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
